ledger/ledger-api-integration-tests/src/test/itsuite/scala/com/digitalasset/platform/tests/integration/ledger/api/ResetServiceIT.scala

```diff
@@ -57,16 +54,13 @@ class ResetServiceIT
       "return a new ledger ID" in allFixtures { ctx =>
         for {
           lid1 <- ctx.ledgerIdentityService.getLedgerIdentity(GetLedgerIdentityRequest())
-          lid1SecondInstance <- ctx.ledgerIdentityService.getLedgerIdentity(
-            GetLedgerIdentityRequest())
+          lid1Bis <- ctx.ledgerIdentityService.getLedgerIdentity(GetLedgerIdentityRequest())
           lid2 <- ctx.reset()
           lid3 <- ctx.reset()
-          throwable <- ctx.resetService.reset(ResetRequest(lid1.ledgerId)).failed
```

Comment:
why did you drop this? it was a valid test case.

Reply:
Done